docs/users/security/safu.md

```diff
@@ -0,0 +1,93 @@
+<!--
+order: 3
+-->
+
+# Simple Arrangement for Funding Upload (SAFU)
+
+Learn about the Simple Arrangement for Funding Upload (SAFU)
+on Evmos {synopsis}
+
+## Overview
+
+<!-- markdown-link-check-disable-next-line -->
+The [Simple Arrangement for Funding Upload (the "SAFU")](https://github.com/evmos/evmos/tree/main/docs/users/security/SAFU.pdf)
+outlines the post-exploit policy for active vulnerabilities
+in the Evmos blockchain. The SAFU is intended for white hat hackers
```

Comment:
nit: semantic line breaks may not look as beautiful but IMO they are better for version control
```suggestion
outlines the post-exploit policy for active vulnerabilities in the Evmos blockchain.
The SAFU is intended for white hat hackers
```